### ISSUE TITLE
Problem with Icon Pack Arcticons

### ISSUE THREAD

**Issue by tuch96** [OpenLauncherTeam/openlauncher#653]
#### General Information

* **OpenLauncher Version:** 0.7.4
* **Android Version:** 10
* **Custom ROM:** LineageOS
* **Install Source:** F-Droid

#### Description

I have a problem with the icon package Arcticons from @Donnnno. The calendar icons are not displayed correctly. The developer of the package has integrated a new dynamic calendar icons function. Since then the problem has been active. He can't find the problem and now I wanted to ask here.

Thanks for your help and reply!!

![111268324-6ae56180-862d-11eb-81a7-b68066c73523](https://user-images.githubusercontent.com/57192210/114301175-e965df80-9ac3-11eb-8fbe-89fab56e24ea.png)

**Comment by dkanada** [comment]
Hi there @Donnnno :smile: Unfortunately I'm not sure why it doesn't work on OpenLauncher, but this app has mostly been in maintenance mode for a few years so it probably won't get solved by us.

**Comment by Donnnno** [comment]
The weird thing is that the fallback should work, but it doesn't. For example, the calendar icon has two variants, the normal static icon and the one that changes every day.

the normal one:
`<item component="ComponentInfo{com.android.calendar/com.android.calendar.AllInOneActivity}" drawable="calendar31" />`
the dynamic one:
`<calendar component="ComponentInfo{com.android.calendar/com.android.calendar.AllInOneActivity}" prefix="calendar" />`

Other launchers support this fallback, but open launcher doesn't for some reason.

**Comment by Donnnno** [comment]
@dkanada Yeah I noticed that already, I'm not using myself, so it isn't a huge problem for me, but I can imagine it being slightly annoying for others. 

I guess the simple solution would be to use a launcher that does support it haha.
